test/k8s-integration/driver.go

```diff
@@ -65,11 +77,18 @@ func installDriver(goPath, pkgDir, stagingImage, stagingVersion, deployOverlayNa
 	if err != nil {
 		return fmt.Errorf("failed to deploy driver: %v", err)
 	}
-
+	klog.Infof("Deploying driver")
+	fmt.Printf("Deploying driver")
```

Comment:
Do we need both klog and fmt?

Reply:
klog result does not show in build-log.txt

Reply:
I think you need to do something like [this](https://github.com/kubernetes-csi/external-provisioner/blob/master/cmd/csi-provisioner/csi-provisioner.go#L111) to get the klog to print to stdout